src/nvim/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 option(USE_GCOV "Enable gcov support" OFF)
 
+if (MSVC)
+  add_compile_options(/nologo)
+endif()
```

Comment:
is that one required ? this file is loaded via `add_subdirectory(src/nvim)` so the option should already be set, shouldnt it ?

Reply:
I'll double check.

Additionally, I'll suppress more of the banners in the third_party directory.